sdk/src/main/scala/com/horizen/account/state/AccountStateView.scala

```diff
@@ -298,9 +304,16 @@ class AccountStateView(
 
   override def getConsensusEpochNumber: Option[ConsensusEpochNumber] = metadataStorageView.getConsensusEpochNumber
 
-  override def getFeePayments(withdrawalEpoch: Int): Seq[BlockFeeInfo] =
-    metadataStorageView.getFeePayments(withdrawalEpoch)
+  override def getFeePayments(withdrawalEpoch: Int, blockToAppendFeeInfo: Option[AccountBlockFeeInfo] = None): Seq[AccountBlockFeeInfo] = {
+    var blockFeeInfoSeq = metadataStorageView.getFeePayments(withdrawalEpoch)
+    blockToAppendFeeInfo.foreach(blockFeeInfo => blockFeeInfoSeq = blockFeeInfoSeq :+ blockFeeInfo)
+
+    if(blockFeeInfoSeq.isEmpty) {
+      return Seq()
+    }
```

Comment:
why do we need this? `blockFeeInfoSeq` is an empty Seq

Reply:
You are right, we do not need it